src/main/scala/firrtl/transforms/BlackBoxSourceHelper.scala

```diff
@@ -135,7 +135,7 @@ class BlackBoxSourceHelper extends Transform with DependencyAPIMigration {
     //  make[1]: *** No rule to make target `test_run_dir/examples.AccumBlackBox_PeekPokeTest_Verilator345491158/AccumBlackBox.v', needed by `.../chisel-testers/test_run_dir/examples.AccumBlackBox_PeekPokeTest_Verilator345491158/VAccumBlackBoxWrapper.h'.  Stop.
     //  or we end up including the same file multiple times.
     if (verilogSourcesOnly.nonEmpty) {
-      writeTextToFile(verilogSourcesOnly.map(_.getCanonicalPath).mkString("\n"), filelistFile)
+      writeTextToFile(verilogSourcesOnly.map(_.getCanonicalPath).mkString("\n") + "\n", filelistFile)
```

Comment:
```suggestion
      writeTextToFile(verilogSourcesOnly.map(_.getCanonicalPath).mkString("", "\n", "\n"), filelistFile)
```

Just an FYI that there is a 3 argument `mkString` that has `start` and `end` arguments as well. Generally it's just a stylistic choice but since that append requires an extra String copy, it could be useful for very large Strings. Probably fine to leave as is in this case though.